demos/tests/cases.py

```diff
@@ -255,7 +255,7 @@ def single_option_cases(key, *args):
             test_cases=combine_cases(
         TestCase(options={'-no_show': None,
             **MONITORS,
-            '-i': TestDataArg('coco128/images/train2017/')}),
+            '-i': DataPatternArg('coco128-every-480x640x3')}),
```

Comment:
This case is closer to the video input (since frame size doesn't change)

Reply:
For some reason there are issues with reading video input in github actions: https://github.com/openvinotoolkit/open_model_zoo/actions/runs/7408585487/job/20157102210?pr=3887

<img width="633" alt="image" src="https://github.com/openvinotoolkit/open_model_zoo/assets/52502732/960a7e75-cf6b-4093-a85f-81e5cc794a6f">

Reply:
You may check OpenCV cmake flags. I heavily reduced build time by disabling modules

Reply:
I see, will check them. Thanks!

Reply:
Done, thanks!